database/object_user_attributes/db.go

```diff
@@ -206,6 +210,46 @@ func (db *DB) GetObjectUserAttributesByObjectAttributeID(
 	return attributes, nil
 }
 
+func (db *DB) GetObjectUserAttributesByObjectIDsAttributeIDs(
+	ctx context.Context,
+	objectAttributeNames []string,
+	objectIDs []umid.UMID,
+	query string,
+	orderType universe.OrderType,
+	limit uint,
+	offset uint,
+) ([]*entry.ObjectUserAttribute, error) {
+	var searchString string
+	if query != "" {
+		searchString = fmt.Sprintf("%%%s%%", query)
```

Comment:
What happens when this input (which comes from user, so considered insecure) already has '%' chars in it?
It's tricky with these sql strings, but the LIKE bit is already done in other function here: https://github.com/momentum-xyz/ubercontroller/blob/develop/database/object_user_attributes/db.go#L727 together with a `ILIKE ... ESCAPE '\\'` to make it a bit more readable and avoid the duplicate '%' meaning